releases/cloud.md

```diff
@@ -15,9 +15,25 @@ Get future release notes emailed to you:
 {% include marketo.html %}
 
 {{site.data.alerts.callout_version}}
-As of November 7, 2022, {{ site.data.products.serverless }} clusters are running CockroachDB [v22.1.10](v22.1.html#v22-1-10) and new {{ site.data.products.dedicated }} clusters are running CockroachDB [v22.1.10](v22.1.html#v22-1-10).
+As of December 5, 2022, {{ site.data.products.serverless }} clusters are running CockroachDB [v22.1.10](v22.1.html#v22-1-10) and new {{ site.data.products.dedicated }} clusters are running CockroachDB [v22.1.10](v22.1.html#v22-1-10).
 {{site.data.alerts.end}}
 
+## December 5, 2022
+
+<h3> Console changes </h3>
+
+- {{ site.data.products.serverless }} clusters now have a [**Metrics** page](../{{site.versions["stable"]}}/ui-custom-chart-debug-page.html) in the Console with charts to **Monitor SQL Activity** and **Identify SQL Problems**.
```

Comment:
I thought this already existed.

Reply:
It never had a release note, so just adding one now. I think it hasn't existed for long.

Reply:
I remember that I including these pages in a [document](https://docs.google.com/document/d/1TK5JZaS8kNTqq_wnQWbzT9MxXaDthV5qboFlCx6CuVU/edit#) that I worked on around September/October, so I don't think they're that new. Would you mind confirming the age of these pages?

Reply:
@thtruo @andf-crl do you know when the metrics page was released for serverless?

Reply:
@gemma-shay Custom metrics were introduced in September for Serverless

Reply:
#[14998](https://github.com/cockroachdb/docs/pull/14998/files) saw the [serverless metrics list](https://www.cockroachlabs.com/docs/stable/ui-custom-chart-debug-page.html?filters=metric-names-serverless); merged from [this PR](https://github.com/cockroachdb/docs/pull/14998/files) published.

This is somewhat similar to the note for log export, which was also introduced to CC previous to the Dec 5 CC release (Aug / Sep '22). I'm not sure how CC does it: but for CRDB I'd only include release notes for changes specific to the version the release applies to, even if previous RNs failed to capture a CC change.

Reply:
We could move it to the September notes, leave it here, or remove the note. I would prefer to leave it in December so users don't miss it, but open to thoughts.

Reply:
@andf-crl whoops I commented before I saw your reply. We don't really attach notes to versions in CC; it's just chronological.